### ISSUE TITLE
Icon for Python Jupyter Notebooks is Wrong

### ISSUE THREAD

**Issue by lvwarren** [googlecolab/colabtools#3219]
When I open up Google Drive in a Google Chrome Browser, latest and greatest, on MacOS, the icon displayed for the file does not consistently correspond with the filetype, in this case .ipynb
<img width="394" alt="Screen Shot 2022-11-08 at 1 33 27 AM" src="https://user-images.githubusercontent.com/589358/200502501-28ac2388-d8ad-44af-b4f5-b7cc72835b91.png">

**Comment by blois** [comment]
The icon in Drive is specified by the mime type of the file which is determined by the application which uploaded the file.

**Comment by lvwarren** [comment]
That is incorrect. It should be determined by the suffix of the file, or else file type becomes a meaningless artifact of whatever application handled it last.

thanks

**Comment by cperry-goog** [comment]
Tracking this internally at b/258678880

.ipynb isn't a Colab specific suffix, so we won't change this for all .ipynb files in Drive, but we can update this for any .ipynb files you interact with in Colab.

Note that these policies are largely a Drive decision, not Colab. They work based on mime type.

**Comment by lvwarren** [comment]
Thanks for your great work in following these issues.

May I suggest that since a major chunk of Colab is the execution of Juypter notebooks, that .ipynb should be incorporated as a Colab specific suffix so that user files are properly annotated with the correct icon type. Otherwise, its an unprofessional glitch, which isn't representative of an otherwise great system.
________________________________
